### ISSUE TITLE
How to test the model on the test set?

### ISSUE THREAD

**Issue by xiyangl3** [alexnowakvila/QAP_pt#1]
I don't see test method in the code.

**Comment by jjanicechen** [comment]
Hi @alexnowakvila, 

Thanks a lot for releasing the code!
I have the same question as above. It seems only the TSP part has test method but not the QAP part?
Would you release that part of code later? Or any tips for us to implement it ourselves?

**Comment by alexnowakvila** [comment]
Hi @xystevenliu , @jjanicechen ,
all my excuses for the delay of my response. As the model was converging with much fewer iterations than the size of the training set, it was not necessary to implement the test mode (I made sure I could ignore it). It is straightforward to implement it yourselves; put is_training=False (change the default) in gen.sample_batch(), and make sure you run over all the test observations (use different batches). A part from that, the function is the same as train() without the optimization lines.

**Comment by xystevenliu** [comment]
Thanks, @alexnowakvila 

It seems that the
